segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/BookKeeperLog.java

```diff
@@ -115,7 +116,8 @@
         this.closed = new AtomicBoolean();
         this.logNodePath = HierarchyUtils.getPath(logId, this.config.getZkHierarchyDepth());
         this.traceObjectId = String.format("Log[%d]", logId);
-        this.writes = new WriteQueue(this.config.getMaxConcurrentWrites());
+        this.writes = new WriteQueue();
```

Comment:
Would it be better to pass the `ConcurrencyManager` instance as a parameter rather than passing values via the `writes` calls?